### PR TITLE
add binary_paintshop example to ionq

### PR DESCRIPTION
**Pull request by MLeib**

### CODE REVIEW

**Comment by balopat** [comment]
Hi @MLeib, I fixed the header and added the _book.yaml entry for our site. I also removed an API key that was left in there, and in order to remove it from git history I rebased your commit. You won't be able to push from your local branch because of this, I recommend removing your local branch (`git branch -D ionq_bpsp_example`) and checkout your remote branch again (`git checkout origin/ionq_bpsp_example -b ionq_bpsp_example` or similar).

File: docs/tutorials/ionq/binary_paintshop.ipynb [hunk 1]
```diff
@@ -0,0 +1,421 @@
+{
```
Comment (dabacon) [inline docs/tutorials/ionq/binary_paintshop.ipynb:1]: <p>If cirq does not install this won't import cirq or ionq.  Suggest</p><pre class="ql-syntax" spellcheck="false">try:
&nbsp; &nbsp; import cirq
except ImportError:
&nbsp; &nbsp; print("installing cirq...")
&nbsp; &nbsp; !pip install --quiet cirq
&nbsp; &nbsp; print("installed cirq.")
import numpy as np
import cirq
import cirq.ionq as ionq
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (dabacon) [inline docs/tutorials/ionq/binary_paintshop.ipynb:1]: <p><code>remote_host</code> is no  longer needed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

**Comment by dabacon** [comment]
A couple of small comments.  I will take another pass on this to flesh it out a little bit more, but let's get this one submitted.

**Review by dabacon — APPROVED** [review]
After fixing the two small details, approval to submit.

**Review by balopat — APPROVED** [review]

**Pull request merged**